modules/lib/lib-event/src/main/resources/lib/xp/event.ts

```diff
@@ -0,0 +1,136 @@
+import * as Lib from './event';
+
+/*
+ * Global library definitions for require and __non_webpack_import__
+ */
+
+type LibType = typeof Lib;
+
+declare global {
+    interface XpEventLibrary extends LibType {}
+}
```

Comment:
Have you considered just doing it like this:

```suggestion
declare global {
    // Will be merged
    interface XpLibraries {
      "/lib/xp/event": typeof import('./event')
    }
}
```

Reply:
Yes, replaced it. Thanks.